src/locales/ko/mystery-encounters/part-timer-dialogue.json

```
@@ -1,31 +1,31 @@
 {
-  "intro": "A busy worker flags you down.",
-  "speaker": "Worker",
-  "intro_dialogue": "You look like someone with lots of capable Pokémon!$We can pay you if you're able to help us with some part-time work!",
-  "title": "Part-Timer",
-  "description": "Looks like there are plenty of tasks that need to be done. Depending how well-suited your Pokémon is to a task, they might earn more or less money.",
-  "query": "Which job will you choose?",
-  "invalid_selection": "Pokémon must be healthy enough.",
+  "intro": "바빠보이는 작업 팀장이 나를 불러세웠다.",
+  "speaker": "작업 팀장",
+  "intro_dialogue": "거기 당신!\n쓸만한 포켓몬들을 가지고 있는거 같네요.$혹시 시간 있으면 알바 안 할래요?",
+  "title": "아르바이트",
+  "description": "해야 할 일들이 산더미같이 쌓여 있네요.\n포켓몬이 일을 얼마나 잘하는지에 따라,\n일당을 더 많이 받을 수도, 더 적게 받을 수도 있을 것 같습니다.",
+  "query": "어떤 아르바이트를 해볼까요?",
+  "invalid_selection": "포켓몬이 건강하지 않습니다.",
   "option": {
     "1": {
-      "label": "Make Deliveries",
-      "tooltip": "(-) Your Pokémon Uses its Speed\n(+) Earn @[MONEY]{Money}",
-      "selected": "Your {{selectedPokemon}} works a shift delivering orders to customers."
+      "label": "배달하기",
+      "tooltip": "(-) 포켓몬의 스피드를 활용\n(+) @[MONEY]{Money} 획득",
+      "selected": "{{selectedPokemon}}[[는]] 배달 아르바이트를 시작했다."
     },
     "2": {
-      "label": "Warehouse Work",
-      "tooltip": "(-) Your Pokémon Uses its Strength and Endurance\n(+) Earn @[MONEY]{Money}",
-      "selected": "Your {{selectedPokemon}} works a shift moving items around the warehouse."
+      "label": "창고정리",
+      "tooltip": "(-) 포켓몬의 힘과 튼튼함을 활용\n(+) @[MONEY]{Money} 획득",
+      "selected": "{{selectedPokemon}}[[는]] 창고 정리를 시작했다."
     },
     "3": {
-      "label": "Sales Assistant",
-      "tooltip": "(-) Your {{option3PrimaryName}} uses {{option3PrimaryMove}}\n(+) Earn @[MONEY]{Money}",
-      "disabled_tooltip": "Your Pokémon need to know certain moves for this job",
-      "selected": "Your {{option3PrimaryName}} spends the day using {{option3PrimaryMove}} to attract customers to the business!"
+      "label": "바람잡이",
+      "tooltip": "(-) {{option3PrimaryName}}의 {{option3PrimaryMove}}를 활용\n(+) @[MONEY]{Money} 획득",
+      "disabled_tooltip": "포켓몬이 해당 작업에 필요한 기술이 없습니다.",
+      "selected": "{{option3PrimaryName}}[[는]] {{option3PrimaryMove}}[[로]]\n하루종일 손님들을 끌어모았다!"
     }
   },
-  "job_complete_good": "Thanks for the assistance!\nYour {{selectedPokemon}} was incredibly helpful!$Here's your check for the day.",
-  "job_complete_bad": "Your {{selectedPokemon}} helped us out a bit!$Here's your check for the day.",
-  "pokemon_tired": "Your {{selectedPokemon}} is worn out!\nThe PP of all its moves was reduced to 2!",
-  "outro": "Come back and help out again sometime!"
+  "job_complete_good": "도와줘서 고마워요!\n오늘은 거의 {{selectedPokemon}} 혼자서 일을 다한 것 같네요!$여기 일당이에요!",
+  "job_complete_bad": "{{selectedPokemon}}[[가]] 뭐… 조금은 도움이 된 거 같네요!$여기 일당이에요.",
+  "pokemon_tired": "{{selectedPokemon}}[[가]] 지친 것 같다!\n모든 기술의 PP가 2까지 줄었다!",
```

```suggestion
  "pokemon_tired": "{{selectedPokemon}}[[가]] 지친 것 같다!\n모든 기술의 PP가 2로 감소했다!",
```
"-까지 줄었다"라고 하면 "1~2만큼 줄었다"라고 잘못 읽힐 수도 있을 듯하여 수정했습니다.